kubernetes-config/locust-worker-controller.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: "extensions/v1beta1"
+kind: "Deployment"
+metadata:
+  name: locust-worker
+  labels:
+    name: locust-worker
+spec:
+  replicas: 5
```

Comment:
oops 😅  mostly just followed by example. what would you recommend for number of workers, type of machine, etc? I have no concept of what we'd need tbh

Reply:
I usually do some trial and error. I can't imagine we need more than 1 worker given our cap is 150QPS. I wonder if it's possible/feasible to use a single container instead of a master/workers (I thought that's what we did on upstream hipstershop)

For machine types, it might be good to see if a single n1-standard-2 works. if not, maybe 2 n1-standard-1's? Or try out a single higher-CPU node. We want to use the minimum resources so we're not wasting customer's money, but you don't need to spend too much time experimenting

Reply:
Ok I basically just reduced everything to 1 and things still work. From my humble noobernetes + locust knowledge, apparently the worker is there for scalability (we can increase the number of replicas / locust workers easily?)

Reply:
Ok sounds good! I don't think we need to worry too much about load generator scalability when the service itself only handles 150QPS, but yeah it's good to have it set up for easy growth later

Reply:
The service itself handles 150qps - this needs to be fixed (not in this PR). The scalability of the loadgen will help us to ramp-up the number and variety of load patterns we can send into the cluster.